### PR TITLE
PR: Update `README.md`, `CONTRIBUTING.md`, screenshot and add `RELEASE.md`

### PR DESCRIPTION
**Pull request by dalthviz**
Part of #59

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+```bash
+$ git clone https://github.com/spyder-ide/spyder-line-profiler.git
+$ cd spyder-line-profiler
+```
+
+### Installing dependencies
+
+This installs Spyder, line_profiler and all other dependencies of the plugin into
+the conda environment using the conda-forge channel.
```
Comment (ccordoba12) [inline CONTRIBUTING.md]:
```suggestion
This installs Spyder, line_profiler and all other plugin dependencies into
the conda environment previously created, using the conda-forge channel.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+You are done! You can run Spyder as normal and it should load the line profiler
+plugin.
+
+```bash
+$ spyder
+```
+
+### Running Tests
+
+This command installs the test dependencies in the conda environment using the conda-forge channel.
```
Comment (ccordoba12) [inline CONTRIBUTING.md]:
```suggestion
This command installs the test dependencies into your conda environment, using the conda-forge channel.
```

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Release
+Follow the steps to release a new version of spyder-line-profiler.
```
Comment (ccordoba12) [inline RELEASE.md]:
```suggestion

Follow these steps to release a new version of spyder-line-profiler.
```

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+You will need to have a local clone of the repo. The following steps supose a repo setup from a fork with and `upstream` remote pointing to the main `spyder-line-profiler` repo
+
+* Close [milestone on Github](https://github.com/spyder-ide/spyder-line-profiler/milestones)
+
+* Update local repo
+
+  ```bash
+  git restore . && git switch master && git pull upstream master
+  ```
+* Clean local repo
```
Comment (ccordoba12) [inline RELEASE.md:21]:
```suggestion

* Clean local repo
```

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+  ```bash
+  python -m pip install --upgrade pip
+  pip install --upgrade --upgrade-strategy eager build setuptools twine wheel
+  ```
+
+* Build source distribution and wheel
+
+  ```bash
+  python -bb -X dev -W error -m build
```
Comment (ccordoba12) [inline RELEASE.md:55]: I think this command won't work because we're not using `pyproject.toml` here. Did you check it?

**Review by ccordoba12 — COMMENTED** [review]
Thanks for this @dalthviz! I left some minor suggestions/comments for you.

Looking at the screenshot: are we using here colors that come from SpyderPalette? If not, I think we should to make things uniform with the rest of Spyder.

File: RELEASE.md (hunk 5 above)
Comment (dalthviz) [reply RELEASE.md:55]: I just copy-pasted from qtpy 😅 although checking seems like the command works: 

imagen

**Comment by dalthviz** [comment]
>Looking at the screenshot: are we using here colors that come from SpyderPalette? If not, I think we should to make things uniform with the rest of Spyder.

I don't think so (that we are currently using the SpyderPalette). I think #50 covers that. Maybe that could be something to do after the 0.3.0 release?

**Comment by ccordoba12** [comment]
> Maybe that could be something to do after the 0.3.0 release?

Ok, no problem. Let's leave that for later.

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz for your work on this!

**Pull request merged**